superset/views/core.py

```diff
@@ -2752,7 +2752,7 @@ def _get_sqllab_payload(user_id: int) -> Dict[str, Any]:
         }
 
     @has_access
-    @expose("/sqllab")
+    @expose("/sqllab", methods=["GET", "POST"])
```

Comment:
this is to make /sqllab page accept POST

Reply:
The request is accepted but the post data is not used, hence the SQL query is actually not pre-populated in the SQL editor as it is now. Previously the search query is acquired at the client side: https://github.com/apache/incubator-superset/commit/cefedf6dffac53924893d7765364c47b65d6fa74#diff-2782bdc5cd0627eef4679c85a44d4ac1L102

Reply:
i merged your code snippet. now it works. thanks!